app/assets/stylesheets/dlme.scss

```diff
@@ -386,4 +400,13 @@ body {
   dd, dl {
     margin-bottom: 0.25rem;
   }
-}
\ No newline at end of file
+}
+
+// Turn off transparency for the topbar on pages where there's no masthead
```

Comment:
Can you add  "(e.g. the pages provided by Devise)"

Reply:
added